model/datacenter/registry.yaml

```diff
@@ -0,0 +1,58 @@
+groups:
+  - id: registry.datacenter
+    stability: experimental
+    type: attribute_group
+    display_name: Datacenter Attributes
+    brief: >
+      Generic DataCenter infrastructure tagging.
+    attributes:
+      - id: dc.name
+        type: string
+        stability: experimental
+        brief: 'Name of datacenter.'
+        examples: ['dc-name']
+      - id: dc.provider.name
```

Comment:
it looks like a duplicate of `cloud.provider` attribute - https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/cloud.md.

Reply:
Indeed I suppose this would fall under "Custom".